actionmailbox/actionmailbox.gemspec

```diff
@@ -38,4 +38,6 @@ Gem::Specification.new do |s|
   s.add_dependency "actionpack",    version
 
   s.add_dependency "mail", ">= 2.7.1"
+
+  s.add_development_dependency "aws-sdk-sns", "~> 1.21"
```

Comment:
We put development dependencies in the Gemfile at the root.

This one’s [already there](https://github.com/rails/rails/blob/598dc9f147e6153a461d6707ca0bf6e356e321ad/Gemfile#L95). I must’ve missed it when I ripped out the old ingress.

```suggestion
```